types/jest/jest-tests.ts

```diff
@@ -1498,6 +1498,32 @@ test.each([[1, 1, 2], [1, 2, 3], [2, 1, 3]])(
     5000
 );
 
+const constCases: [['a', 'b', 'ab'], ['d', 2, 'd2']] = [['a', 'b', 'ab'], ['d', 2, 'd2']];
```

Comment:
You can just do `declare const`, since these are compilation tests, so there’s no need for code duplication:

```suggestion
declare const constCases: [['a', 'b', 'ab'], ['d', 2, 'd2']];
```